update 1

HABT29 opened the pull request:
Added vignette, experimented with different courts, not ready

HABT29 committed:
update 1

Added vignette, experimented with different courts, not ready

HABT29 committed:
update 2

Roy-M-Moore reviewed (commented):
The vignette outline looks good and is correctly placed in a vignettes folder. I like how you have included a lot of screenshots so users can understand what the app is showing them and how it works. You might want to add more instructions for the installation for the final version.

HABT29 committed:
update 3

HABT29 committed:
update 4

Different courts work now, play by play function changes the players image,
some general rearrangements

veerlevijverberg reviewed (approved):
Great job on your vignette!!! I like the use of screenshots. Overall, it provides a clear and concise introduction to the NBA play-by-play functionality and showcases the main features of the app. 

As an overview, you may want to add a brief explanation of the benefits of visualizing the whole game, e.g. gaining insights into scoring patterns, shot selection, or team performance throughout the game.
